test/screenshot/mdc-icon-button/fixture.scss

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+.test-main--icon-button {
+  width: 348px; // fits 4 columns of icon buttons within a Galaxy S7 viewport
+}
+
+.test-cell--icon-button {
+  width: 81px;
```

Comment:
The ripples for the icon buttons on icon-size.html are cut off at the bottom and right with this size. 91 fits those ripples.